screens/EpisodeInputScreen.tsx

```diff
@@ -0,0 +1,398 @@
+      setEpisodes(episodes);
+      resetEpisodeInput();
+    }
+  };
+
+  const resetEpisodeInput = () => {
+    setEpisodeName("");
+    setInitials("");
+    setStartTime(undefined);
+    setEndTime(undefined);
+  };
+  const validateEpisode = () => {
+    return (
+      episodeName !== "" && startTime !== undefined && endTime !== undefined
+    );
+  };
+
+  const hasError = !validateEpisode();
+  const onMount = async () => {
+    const startDay = await AsyncStorage.getItem(STORAGE_KEYS.start_day());
+    const todayEpisodes = await AsyncStorage.getItem(
+      STORAGE_KEYS.daysEpisodes(recordingDay)
+    );
+    if (startDay === null) {
+      await AsyncStorage.setItem(STORAGE_KEYS.start_day(), CURRENT_DATE);
+    } else {
+      const msApart =
+        new Date(CURRENT_DATE).getMilliseconds() -
+        new Date(startDay).getMilliseconds();
+      recordingDay = msApart / DAY_IN_MS + 1;
+    }
+    if (todayEpisodes === null) {
+      await AsyncStorage.setItem(
+        STORAGE_KEYS.daysEpisodes(recordingDay),
+        JSON.stringify([])
+      );
+    } else {
+      setEpisodes(JSON.parse(todayEpisodes));
+    }
+  };
+
+  useEffect(() => {
+    onMount();
+  }, []);
+
+  return (
+    <View style={containerStyles.container}>
+      <View style={containerStyles.titleContainer}>
+        <Text style={styles.title}>Create an episode</Text>
+        <Icon
+          name="three-bars"
+          size={30}
+          color="#000"
+          onPress={() => {
+            navigation.navigate("EpisodeDisplay", { episodes });
+          }}
+        />
+      </View>
+      <View style={containerStyles.inputContainer}>
+        <View style={{ flex: 0.5 }} />
+        <View style={styles.input}>
+          <Text style={styles.inputHeader}>Episode Name</Text>
+          <TextInput
+            style={{ ...styles.inputContent, ...styles.textInput }}
+            maxLength={70}
+            onChangeText={setEpisodeName}
+            value={episodeName}
+            editable
+          />
+        </View>
+        <View style={styles.input}>
+          <Text style={styles.inputHeader}>{"Duration"}</Text>
+          <View style={{ ...styles.inputContent, ...styles.timePickers }}>
+            <TimePicker
+              style={styles.timePickerContainer}
+              time={startTime}
+              isVisible={isStartDatePickerVisible}
+              label="Start time"
+              setTime={setStartTime}
+              setVisibility={setStartDatePickerVisible}
+            />
+            <View style={{ flex: 10 }} />
+            <TimePicker
+              style={styles.timePickerContainer}
+              time={endTime}
+              isVisible={isEndDatePickerVisible}
+              label="End time"
+              setTime={setEndTime}
+              setVisibility={setEndDatePickerVisible}
+            />
+          </View>
+        </View>
+        <View style={styles.input}>
+          {/* <View style={{ flex: 1, flexDirection: "row" }}> */}
```

Comment:
The power of the `Text` inside of `Text` (also looks like the commented out section should be deleted.

Reply:
YEET